eng/build.ps1

```diff
@@ -573,6 +573,7 @@ function Setup-IntegrationTestRun() {
     Capture-Screenshot $screenshotPath
   }
 
+  $env:ServiceHubTraceLevel = "Information"
```

Comment:
How could the log be viewed? Do we need to log to the cloud run machine to see it?

Reply:
Diagnostics level might generate a lot of log so it might be harmful. But just curious how dangerous it is?